src/packages/@support/features/release-notes/layout/ReleaseNotes.svelte

```diff
@@ -24,7 +24,7 @@
   /**
    * External URL for Sparrow's GitHub page.
    */
-  const externalSparrowRealseNote = constants.SPARROW_DOWNLOAD_LINK;
+  const externalSparrowRealseNote = constants.VITE_SPARROW_GITHUB;
```

Comment:
it will break the flow, take SPARROW_GITHUB from constants, their is no VITE_SPARROW_GITHUB export member in constants.

Reply:
Resolved